File: 1 - Iniciante/ex1070.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int i, impar = 0, cont = 6;

    /*entrada*/
    scanf("%d", &i);

    /*processamento & saída*/
    while(impar < cont){
        if(i % 2 == 1){
            printf("%d\n", i);
            impar++;
        }
        i++;
    }
    return 0;
}

File: 1-Iniciante/ex1070.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd

// processamento & saida
let x = parseInt(input);
if(x % 2 === 0) x++;
for(let i = x, cont = 0; cont < 6; i += 2, cont++) console.log(i);